examples/robust_vision_benchmark/README.md

```diff
@@ -0,0 +1,39 @@
+## CleverHans Example Submission to the Robust Vision Benchmark
+
+Using the wrappers in this repository, submitting a CleverHans attack to the [Robust Vision Benchmark](https://robust.vision/benchmark) requires just a few lines of code:
+
+```python
```

Comment:
Could you remove the code snippet from the readme and just point to the file in the `cleverhans_attack_example` folder? We've had issues in the past with example code copied in markdown files getting out-dated.

Reply:
Can you address this? We can merge afterwards

Reply:
Done